docs/docs/reaching-out-to-user.mdx

```diff
@@ -0,0 +1,459 @@
+
+
+## External Events
+
+Sometimes you want an external device to change the course of an ongoing conversation.
+For example, if you have a moisture-sensor attached to a Raspberry Pi, you could use it to notify
+you when a plant needs watering via your assistant.
+
+The examples below are from the [reminderbot example bot](https://github.com/RasaHQ/rasa/blob/master/examples/reminderbot),
+which includes both reminders and external events.
+You can clone it and follow the instructions in `README` to try out the full version.
+
+### Using the `trigger_intent` Endpoint
+
+To have an event from an external device change the course of an ongoing conversation, you can
+have the device post to the
+[`trigger_intent` endpoint](./http-api.mdx#operation/triggerConversationIntent) of your conversation.
+The `trigger_intent` endpoint injects a user intent (possibly with entities) into your conversation.
+For Rasa, it is as if you entered a message that got classified with that specific intent and entities.
+The assistant will then predict and execute the next action as usual.
+
+
+For example, the following post request would inject the intent `EXTERNAL_dry_plant` and the `plant` entity
+into the conversation with id `user123`:
+
+```shell
+curl -H "Content-Type: application/json" -X POST \
+  -d '{"name": "EXTERNAL_dry_plant", "entities": {"plant": "Orchid"}}' \
+  "http://localhost:5005/conversations/user123/trigger_intent?output_channel=latest"
+```
+
+#### Getting the Conversation ID
+
+To have an external device post to the `trigger_intent` endpoint, you need the conversation ID of the ongoing conversation.
+To try it out locally, you can define a custom action `action_tell_id` to tell you
+the conversation ID, and then use it in the POST request to the `trigger_intent` endpoint:
+
+```python
+class ActionTellID(Action):
+    """Informs the user about the conversation ID."""
+
+    def name(self) -> Text:
+        return "action_tell_id"
+
+    async def run(
+        self, dispatcher, tracker: Tracker, domain: Dict[Text, Any]
+    ) -> List[Dict[Text, Any]]:
+
+        conversation_id = tracker.sender_id
+
+        dispatcher.utter_message(
+            f"The ID of this conversation is: " f"{conversation_id}."
+        )
+
+        dispatcher.utter_message(
+            f"Trigger an intent with "
+            f'curl -H "Content-Type: application/json" '
+            f'-X POST -d \'{{"name": "EXTERNAL_dry_plant", '
+            f'"entities": {{"plant": "Orchid"}}}}\' '
+            f"http://localhost:5005/conversations/{conversation_id}/"
+            f"trigger_intent"
+        )
+
+        return []
+```
+
+In a real-life scenario, your external device would get hold of the conversation ID some other way.
+In the dry plant example, you might have a database of plants, the users that water them, and the users'
+conversation IDs. Your Raspberry Pi would get the conversation ID directly from the database.
```

Comment:
i'm kind of confused about this whole section, what is it supposed to illustrate exactly? it doesn't actually do anything that's relevant to triggering actions does it?

Reply:
i'm going with removing it and adding a link to the readme